infra/src/dsl/types/input-types.ts

```diff
@@ -28,7 +28,6 @@ export type BffInfo = {
   globalPrefix: string
   env?: EnvironmentVariables
   secrets?: Secrets
-  allowedRedirectUris?: Partial<Record<OpsEnvWithLocal, string[]>>
```

Comment:
~Kind of wish there was no PortalKeys magic inside the bff() function, and everything important was passed in (scopes and redirect uris. But we probably won't add more bff's, so whatever :)